src/test/scala-2.11/rx/lang/scala/completeness/ObservableCompletenessKit.scala

```diff
@@ -198,8 +198,8 @@ class ObservableCompletenessKit extends CompletenessKit {
     // manually added entries for Java static methods
     "amb(Iterable[_ <: Observable[_ <: T]])" -> "amb(Observable[T]*)",
     "create(OnSubscribe[T])" -> "apply(Subscriber[T] => Unit)",
-    "create(SyncOnSubscribe[S, T])" -> "[TODO]",
-    "create(AsyncOnSubscribe[S, T])" -> "[TODO]",
+    "create(SyncOnSubscribe[S, T])" -> "create(SyncOnSubscribe[S, T])",
+    "create(AsyncOnSubscribe[S, T])" -> "create(AsyncOnSubscribe[S, T])",
```

Comment:
Can't you just delete these two lines? This list should only mention correspondences where Scala differs from Java, but here the signatures are the same.